apps/estuary/lib/estuary_web/router.ex

```diff
@@ -26,6 +26,7 @@ defmodule EstuaryWeb.Router do
 
     get("/", Redirect, to: "/events")
     live("/events", EventLiveView)
+    live("/streaming-events", StreamingEventLiveView, layout: {EstuaryWeb.LayoutView, :root})
```

Comment:
You might consider adding the `layout` part to the events view too, as I "think" that will be missing its styling if you don't.